### PR TITLE
Shape shift

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

File: content/attacks/posts/2016-03-14-ShapeShift.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+date: 2016-03-14
+target-entities: ShapeShift
+entity-types: Exchange
```
Comment (JediFaust) [inline content/attacks/posts/2016-03-14-ShapeShift.md:4]: Add `Custodian`

File: content/attacks/posts/2016-03-14-ShapeShift.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  - Insider Attack
+  - Infrastructure Attack
+  - Wallet Hack
+title: "ShapeShift Exchange Hacked for $230000"
+loss: 230000
+---
+
+## Summary
+
+From March 14 to April 9, 2016, there were 3 hacks of the Swiss ShiftShape crypto exchange.
```
Comment (JediFaust) [inline content/attacks/posts/2016-03-14-ShapeShift.md:15]: fix `3 hacks` to `three hacks` and do so on everywhere

File: content/attacks/posts/2016-03-14-ShapeShift.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+The first hack was made by an employee of the exchange. Then he left a backdoor in the system and sold access to a hacker, who committed two more hacks.
```
Comment (JediFaust) [inline content/attacks/posts/2016-03-14-ShapeShift.md:16]: In the first sentence, I guess the word `hack` is unappropriate, and please provide more details, for example `stole the keys of exchange's hot wallet` or something. The second sentence is also can be extended with technical details.

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Comment by JediFaust** [comment]
> `...more hacks. Attackers Behind the attack was an exchange employee, his name has not ...` Issue: Incorrect form of “be” Fix: `was` to `were`
> 
> `...sider fraud: An exchange employee stole cryptocurrency and left a backdoor in the system. Weak...` Issue: Missing article Fix: `cryptocurrency` to `a cryptocurrency`

Please, fix grammar issues above

**Comment by Williamalong23** [comment]
✅ Verified in all 🇱🇷 services, Recover Bitcoin, iCloud bypass, Bank loader, game hacks ,Wi-Fi recovery ,Instagram , Facebook ,Snapchat , Twitter ,Gmail, cashapp, PayPal, and other hacking job👌Contact Him 22 on Instagram for quick recovery now https://instagram.com/22cracker?igshid=YWYwM2I1ZDdmOQ==

**Comment by svg-arch** [comment]
@JediFaust

**Review by evgenydmitriev — APPROVED** [review]
@svg-arch, when you submit articles, it would help to run your English text by GPT to make the writing style look more professional. Here's an example of a GPT prompt that might help, but feel free to experiment with it:

> Rewrite the text below in a more professional engineering voice using IT security industry vocabulary.

**Pull request merged**